frontend/public/locales/en/translation.json

```diff
@@ -2130,6 +2139,7 @@
   "Prerequisites and Configuration": "Prerequisites and Configuration",
   "Press space or enter to begin dragging, and use the arrow keys to navigate up or down. Press enter to confirm the drag, or any other key to cancel the drag operation.": "Press space or enter to begin dragging, and use the arrow keys to navigate up or down. Press enter to confirm the drag, or any other key to cancel the drag operation.",
   "preview.clusterPools": "<bold>Technology Preview</bold>: For more information, <a>view documentation</a> on Cluster pools",
+  "preview.discoveredPolicies": "<bold>Technology Preview</bold>: For more information, <a>view documentation</a> on Discovered policies",
```

Comment:
We technically do not have a topic about Discovered policies. Can you remind me what Discovered policies are? Last I recall, discovered policies are external policies. I should've mentioned that there is a `DiscoveryConfig` resource for MCE: https://github.com/stolostron/rhacm-docs/blob/2.12_stage/clusters/discovery/discovery_config_cli.adoc#discovery-enable-cli

We should probably reconsider using "Discovered policies"

Reply:
Let's remove link and just show that it is tech preview. How about "<bold>Technology Preview</bold>: Discovered policies"

Reply:
@mprahl

Reply:
Is there a story or feature for this tech preview item? I haven't seen a doc issue for it.

Reply:
We don't have one for now. https://issues.redhat.com/browse/ACM-12484